crmc.sh

```diff
@@ -16,9 +16,22 @@ case $ARCHITECTURE in
   ;;
 esac
 
+# In case root is available determine std
```

Comment:
ROOT is not a dependency of `CRMC`, AFAICT. If it can be, it should be properly declared. Moreover, why do you set only `c++XX`? Would't it make sense to have something like:

```
STD="-std=c++XX"
```

? Also I  think you can cleanup a bit the if condition to make it more linear.